Backend/controller/PatientController.go

```diff
@@ -0,0 +1,84 @@
+package controller
+
+import (
+	"api/models"
+	"api/service"
+	"encoding/json"
+	"log"
+	"net/http"
+	"strconv"
+
+	"github.com/go-chi/chi/v5"
+)
+
+func GetPatientById(w http.ResponseWriter, r *http.Request) {
+	id, err := strconv.Atoi(chi.URLParam(r, "id"))
```

Comment:
Verificar se o erro é a falta de id pelo usuário, caso for, o erro é 400, o if ali em baixo só da 500